ports/atmel-samd/boards/matrixportal_m4/pins.c

```diff
@@ -30,6 +66,9 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_OBJ_NEW_QSTR(MP_QSTR_MOSI),MP_ROM_PTR(&pin_PA19) },
     { MP_OBJ_NEW_QSTR(MP_QSTR_MISO),MP_ROM_PTR(&pin_PA17) },
 
+    { MP_OBJ_NEW_QSTR(MP_QSTR_MTX_ADDRESS),MP_ROM_PTR(&matrix_addr_tuple) },
```

Comment:
`addr` is used in the arg list, so we can use the abbreviation to be consistent.
```suggestion
    { MP_OBJ_NEW_QSTR(MP_QSTR_MTX_ADDR_PINS),MP_ROM_PTR(&matrix_addr_tuple) },
```

Reply:
is it ok if five pins are given but only four are needed? Does Protomatter ignore the fifth pin?

Reply:
no, that's why you have to "slice" the address pin list if you actually use fewer address pins.